### PR TITLE
Aaf 63 models

### PR DESCRIPTION
**Pull request by phuchle**
Merging this into dev so it doesn't become any bigger than it is right now.  We'll be able to branch from dev after this. 

Maybe close the original issue since it is too broad.

### CODE REVIEW

**Comment by rainrivas** [comment]
@phuchle we need to fix the bcrypt stuff. I can run update on it and just push the update but no guarantees that the bcrypt stuff will work

**Review by jwu910 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -23,25 +26,32 @@
...
-    "eslint-plugin-react": "^7.4.0",
+    "concurrently": "^3.5.1",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-to-json": "^3.3.1",
+    "eslint": "^4.17.0",
+    "eslint-plugin-react": "^7.6.1",
+    "jest": "^22.2.2",
+    "jest-cli": "^22.2.2",
     "node-sass-chokidar": "0.0.3",
```
Comment (jwu910) [inline package.json]: I think we need to update this to a newer version. Node sass failing to build with node-gyp

**Review by jwu910 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (rainrivas) [reply package.json]: yeah i was about to push up a change to travis file to ignore node latest @jwu910

**Review by rainrivas — APPROVED** [review]
Welp, it's working it. Lets get it into dev and start clearing the rest of the branches so we cna decide what we want to do next.

**Review by jwu910 — APPROVED** [review]
:fire:

**Comment by jwu910** [comment]
I'd say lets wait on deleting branches cause i know we had some work in progress that haven't been merged in, we can bring those up to date, and see if we can get PRs out for those

**Comment by rainrivas** [comment]
Why wait? Squash > Merge > Delete

**Comment by rainrivas** [comment]
when i said "clearing rest of branches" I meant by way of PRs

**Comment by jwu910** [comment]
Oh, i meant the other branches

**Comment by jwu910** [comment]
:+1:

**Pull request merged**